example/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": ["react-app"],
+    "rules": {
+        "no-console": "error"
+    }
+}
```

Comment:
I needed to copy this into the `example` subdirectory because the Netlify build runs with the `example` directory as its base, and won't pick up `.eslintrc` in the parent directory.

Reply:
I've removed this since we're building from the repository root now and ESLint is able to detect it during the build.